services/port_manager/src/main/java/com/futurewei/alcor/portmanager/processor/NeighborProcessor.java

```diff
@@ -23,8 +23,7 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
-@AfterProcessor(PortProcessor.class)
-public class NeighborProcessor extends AbstractProcessor {
+public class NeighborProcessor {
```

Comment:
Oh why not inherit from AbstrateProcessor?  @DavidLiu506

Reply:
Port manager will create process chain use AbstrateProcessor.
Set<Class<? extends AbstractProcessor>> subClasses = ReflectionUtil.getSubClassByInterface(
                "com.futurewei.alcor.portmanager.processor", AbstractProcessor.class);